docs/getting-started/tools/index.md

```diff
@@ -20,8 +20,8 @@ environment.
 
 - [Visual Studio Code](https://code.visualstudio.com/) - used for all Typescript projects.
   Suboptimal for C#. Be sure to install [extensions](#visual-studio-code-extensions)
-- [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/) - Pretty good at C# (Server
-  and Mobile)
+- [JetBrains Rider](https://www.jetbrains.com/rider/download/) - fully featured IDE for C#, .NET &
+  more.
```

Comment:
I think this is worth mentioning explicitly just to make sure we're doing the right thing as a commercial user:

```suggestion
- [JetBrains Rider](https://www.jetbrains.com/rider/download/) - fully featured IDE for C#, .NET &
  more. Bitwarden developers should contact IT for a license.
```